qa/L0_infer/test.sh

```diff
@@ -39,7 +39,6 @@ export CUDA_VISIBLE_DEVICES=0
 
 CLIENT_LOG_BASE="./client"
 INFER_TEST=infer_test.py
-EXPECTED_NUM_TESTS=${EXPECTED_NUM_TESTS:="42"}
```

Comment:
It seems like we should keep this so that we have a default value for EXPECTED

Reply:
It is unnecessary since we have that case covered here:
https://github.com/triton-inference-server/server/blob/605d4aa72b80f3289a4078813fcb91d02fa2ce1d/qa/L0_infer/test.sh#L63-L67